client/packages/common/src/intl/locales/en/common.json

```diff
@@ -1362,7 +1363,7 @@
   "label.visit-start": "Start",
   "label.volume-per-outer-pack": "Volume per outer pack",
   "label.volume-per-pack": "Volume per pack",
-  "label.volume-per-unit": "Volume per unit (L)",
+  "label.volume-per-unit": "Volume per unit (m³)",
```

Comment:
How will we transition existing setups? Won't think break all the GAPS datasets?